DataFormats/L1TMuonPhase2/src/classes_def.xml

```diff
@@ -0,0 +1,28 @@
+<lcgdict>
+ <selection>
+
+ <class name="l1t::MuonStub" ClassVersion="4">
+   <version ClassVersion="4" checksum="1164456369"/>
+   <version ClassVersion="3" checksum="939575486"/>
```

Comment:
Why start with version 4? The checksum fore version 3 is the same as for `l1t::TrackerMuon` and `l1t::SAMuon`, which looks suspicious.